spec/services/mixpanel_service_spec.rb

```diff
@@ -919,5 +921,53 @@ def inst_test
         )
       )
     end
+
+    context "when dropping events" do
+      before do
+        routes.draw { get "index" => "anonymous#index" }
+      end
+
+      context "for those not allowed" do
+        def self.test_security_metrics_ips
+          security_metrics_subnet = IPAddr.new("192.211.152.0/24") # 255 IP addresses in a Class C subnet
+          ip_addresses = security_metrics_subnet.to_range
```

Comment:
This is a bit blunt of an approach because it's testing _every_ IP address but it's a peace of mind that I'm comfortable with. Advise!

Reply:
I'd be happier just having one example of an IP in the subnet which gets filtered, and one example of an IP outside of the subnet where the mixpanel event is still sent. Testing every IP in the subnet doesn't tell us anything new, just whether `IpAddr.new ... include?` is working correctly

Reply:
Gotcha! That makes sense to me